app/services/claims/provider/generate_csv.rb

```diff
@@ -1,6 +1,6 @@
 require "csv"
 
-class Claims::Provider::GenerateCsv
+class Claims::Provider::GenerateCSV
```

Comment:
Could this inherit from `ApplicationService`?

Reply:
I've moved the `ApplicationService` introduction into https://github.com/DFE-Digital/itt-mentor-services/pull/984.